src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx

```diff
@@ -1,6 +1,9 @@
-import { cleanup, render } from '@testing-library/react'
+import { cleanup, render, screen } from '@testing-library/react'
 import { shallow } from 'enzyme'
+import * as path from 'path'
+import * as fs from 'fs'
```

Comment:
I think we can use the default import here:
```suggestion
import path from 'path'
import fs from 'fs'
```